pulp_rpm/app/downloaders.py

```diff
@@ -84,7 +84,7 @@ async def _run(self, extra_data=None):
         else:
             url = self.url
 
-        async with self.session.get(url) as response:
+        async with self.session.get(url, proxy=self.proxy, proxy_auth=self.proxy_auth) as response:
```

Comment:
I see that in the core we pass `auth` and not `proxy_auth`. https://github.com/pulp/pulpcore/blob/master/pulpcore/download/http.py#L209
I'm not familiar enough to say which one is right. Should both be passed? so if proxy is set, then `proxy_auth` is used?

Reply:
Pulpcore is doing it wrong.

Reply:
I am not sure about auth, as I followed aiohttp docs https://docs.aiohttp.org/en/stable/client_advanced.html#proxy-support

Reply:
we don't use proxy_auth alone, so changing as in the core for basic http authentication